### ISSUE TITLE
[Accessibility- Color 3.1] Incorrect aria-label rendering

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3759]
### Problem Statement

Environment: Tribe

#### Steps to Reproduce
1. Create an article then publish it in the stream
2. Like it
3. Comment it
4. Send a kudo on it
5. Share ir

#### Current behavior
  The aria-labels of Like, comment, Send a kudo and Share icons are incorrectly rendred

#### Expected behavior 
The aria-labels should be rendred as follow  "You liked", "You commented", "You sent a kudos" and "You shared"
PS: This is a regression of  [this task
](https://builders.meeds.io/portal/mycraft/dashboard/tasks/taskDetail/8932)
